platforms/core-runtime/internal-instrumentation-processor/src/main/java/org/gradle/internal/instrumentation/extensions/property/PropertyUpgradeClassSourceGenerator.java

```diff
@@ -216,7 +223,7 @@ private static CodeBlock getDeprecationCodeBlock(PropertyUpgradeRequestExtra req
             .build();
     }
 
-    private static CodeBlock getGetCall(String propertyGetterName, CallableReturnTypeInfo returnType, GradleLazyType upgradedPropertyType) {
+    private static CodeBlock getGetCall(String propertyGetterName, PropertyUpgradeRequestExtra implementationExtra, CallableReturnTypeInfo returnType, GradleLazyType upgradedPropertyType) {
```

Comment:
💅 `getGetCall()` sounds a bit weird. What we are doing here is we are generating a getter body. WDYT about `generateGetterBody()` and `generateSetterBody()`?

Reply:
Makes sense, fixed